sotodlib/tod_ops/fft_ops.py

```diff
@@ -454,3 +454,76 @@ def fit_noise_model(
     if merge_fit:
         aman.wrap(merge_name, noise_fit_stats)
     return noise_fit_stats
+
+
+def build_filter_params_dict(
```

Comment:
Name changed. Regarding the `filters.get_hpf` function, my current understanding is that what happens in `processes.FourierFilter` (i.e. the definition of the filter) is already a generalization of the `get_hpf` and `get_lpf` functions (since the type of filter to use is defined by the filter name used in the configuration file).
One option would be to externalize the filter definition done in `processes.FourierFilter` to a separate function (a new one or even the `build_hpf_params_dict` which would be then called `build_filter` or something similar)